templates/password.json

```diff
@@ -4,13 +4,27 @@
     "main": {
       "type": "newsletter",
       "settings": {
-        "heading": "Opening soon",
-        "subheading": "<p>Be the first to know when we launch.</p>",
         "full_width": false
-      }
+      },
+      "blocks": {
+        "heading": {
+          "type": "heading",
+          "settings": {
+            "heading": "Opening soon"
+          }
+        },
+        "subheading": {
+          "type": "paragraph",
+          "settings": {
+            "paragraph": "<p>Be the first to know when we launch.</p>"
+          }
+        },
+        "email": {
+          "type": "email_form"
+        }
+      },
+      "block_order": ["heading", "subheading", "email"]
     }
   },
-  "order": [
-    "main"
-  ]
+  "order": ["main"]
```

Comment:
Can you preserve the formatting? I assume you have auto-formatting on in VS Code.